settings.c

```diff
@@ -889,12 +926,28 @@ bool settings_override_acceleration (uint8_t axis, float acceleration)
     } else {
         if(!override_backup.valid)
             save_override_backup();
-        settings.axis[axis].acceleration = acceleration * 60.0f * 60.0f; // Limit max to setting value?
+        settings.axis[axis].acceleration = (override_backup.acceleration[axis] >= (acceleration * 60.0f * 60.0f)) ? (acceleration * 60.0f * 60.0f) : override_backup.acceleration[axis]; // Limited to max setting value
     }
-
     return true;
 }
 
+#if ENABLE_ACCELERATION_PROFILES
+ActiveAccelProfile = 1; // Initialize machine with 100% Profile
+
+//Acceleration Profiles for G187 P[x] in percent of maximum machine acceleration.
+float LookupProfile(uint8_t Profile) {
```

Comment:
Move to planner.c as static inlined?
and the return value is either 0 or 1 since you assign the value to Profile before returning it? Use return lookup(profile); instead?
BTW I do not use camel cased or upper case names in the core.

Reply:
Moving it to planner.c as static inlined would make it very hard to find if you wanted to have other profiles instead of the generic placeholder 20% ones. 

I shied away from making it yet another list of $ settings because it would likely mean a whole slew of settings for however much profiles you wanted. having the lookuptable in settings.c makes it findable though. 

maybe move it to the driver my_machine.h because it is a compile time setting?